scripts/get_message_templates.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+path="$1"
+function getMessageTemplates() {
+  grep -h -A 1 -R "\\.withMessageTemplate" --include="*.java" --exclude="*Test*" "$path" | pcregrep -M "\.withMessageTemplate\((.|\n)*?\)" | awk '{ if (prev) $0 = prev " " $0; if ($0 ~ /[,+]$/) { prev = $0; next } prev = ""; print $0 } END { if (prev) print prev }' | awk -F'"' '{sub(/[ \t]+$/, "", $2); sub(/[ \t]+$/, "", $3); sub(/^[ \t\r\n]+/, "", $4); sub(/^[ \t\r\n]+/, "", $6); print $2, $3, $4, $6}' | sed 's/,/=/' > temp.properties
```

Comment:
Can you add a `\` and a new line after each `|`?
It will improve the readability a **little** bit.